app/scripts/directives/updateTimestamp.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name oncokb.directive:updateTimestamp
+ * @author Jing Su created on 2019/06/26
+ * @description
+ * # refresh date
+ */
+angular.module('oncokbApp')
+    .directive('updateTimestamp', function(_, $timeout, mainUtils, DatabaseConnector, FirebaseModel, $rootScope, firebaseConnector) {
+        return {
+            templateUrl: 'views/updateTimestamp.html',
+            restrict: 'E',
+            scope: {
+                type: '=',
+                hugoSymbol: '=',
+                key: '=',
+                obj: '=',
+                disabled: '=',
+                updateTime: '=',
+                updatedBy: '=',
+                path: '='
+            },
+            replace: false,
+            controller: function($scope) {
+                $scope.clicked = false;
+                $scope.getIconClass = function(time) {
+                    if ($scope.disabled) return 'disabled';
```

Comment:
I saw you've frequently used this format. I would recommend using `ternary operator` instead of the simple if/else. Makes code looks cleaner and easier to understand.

Reply:
Updated